content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -68,10 +68,19 @@ of the form `auto-generated-name.my-svc.my-namespace.svc.cluster-domain.example`
 
 ### A/AAAA records
 
-Any pods created by a Deployment or DaemonSet have the following
-DNS resolution available:
+In general a pod has the following DNS resolution:
 
-`pod-ip-address.deployment-name.my-namespace.svc.cluster-domain.example.`
+`pod-ip-address.my-namespace.pod.cluster-domain.example`.
+
+For example, if a pod in the `default` namespace has the IP address 172.17.0.3, 
+its DNS name is:
+
+`172-17-0-3.default.pod.cluster.local`.
+
+Any pods created by a Deployment or DaemonSet exposed by a service have the 
```

Comment:
nit:
```suggestion
Any pods created by a Deployment or DaemonSet exposed by a Service have the 
```
however, does _deployment-name` make sense for a DaemonSet? Should it be `service-name` instead, or some other detail taken from the Service object?